src/api/routes/emissionsAssessment.ts

```diff
@@ -0,0 +1,114 @@
+import { FastifyInstance } from 'fastify'
+import { z } from 'zod'
+import { prisma } from '@/lib/prisma'
+import type { ReportingPeriodData, CompanyWithEmissions } from '@/jobs/emissions/types'
+import { companyWithEmissionsInclude } from '@/jobs/emissions/types'
+import { assessmentResultSchema, errorResponseSchema } from '@/jobs/emissions/schema'
+import { assessEmissions } from '@/jobs/emissions/assess'
+
+const emissionsAssessmentSchema = z.object({
+  wikidataId: z.string(),
+  years: z.array(z.string())
+})
+
+const getCompanyEmissionsDataFromDatabase = async (wikidataId: string, years: string[]): Promise<CompanyWithEmissions | null> => {
```

Comment:
I think we already have this method somewhere?

Reply:
we have none that filters by years so i think it can stay for now, but I'll keep it in mind!